bioblend/_tests/test_util.py

```
@@ -23,6 +29,18 @@ def skip_unless_cloudman():
         return lambda f: f
 
 
+def skip_unless_toolshed():
+    """Decorate tests with this to skip the test if a URL for a ToolShed
+    to run the tests is not provided.
+    """
+    if "BIOBLEND_TOOLSHED_URL" not in os.environ:
+        return unittest.skip(NO_TOOLSHED_MESSAGE)
+    elif requests.get(os.environ["BIOBLEND_TOOLSHED_URL"]).status_code != 200:
+        return unittest.skip(INACCESSIBLE_TOOLSHED_MESSAGE)
```

I think it's better if the tests fail instead of being skipped in this case?
```suggestion
```